package.json

```diff
@@ -23,17 +23,17 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-modals#readme",
   "peerDependencies": {
-    "@angular/animations": ">=4.3.6",
-    "@angular/common": ">=4.3.6",
-    "@angular/core": ">=4.3.6",
-    "@angular/router": ">=4.3.6",
-    "@skyux/core": "^3.1.0",
+    "@angular/animations": "^4.3.6",
```

Comment:
We need to keep the Angular `>=` range in place (that was recently added to support SKY UX 3 work).

Reply:
Oh. I think that was a merge issue. That file conflicted and I think I may have just clicked to use mine and overlooked that there were changes other than the `core` version bump. Fixed.